teacher.rb

```diff
@@ -3,8 +3,8 @@
 class Teacher < Person
   attr_accessor :specialization
 
-  def initialize(name = "Unknown", age = 0, parent_permission = true, specialization = "Unknown")
-    super(name, age, parent_permission)
+  def initialize(name = 'Unknown', specialization = 'Unknown', parent_permission: true)
+    super(name, parent_permission)
     @specialization = specialization
   end
```

Comment:
- [ ] Please pass age in the `initializer` method and in the `super` method as parameter. So while creating teachers users should be able to provide their age.